elements/pl-hidden-hints/pl-hidden-hints.py

```diff
@@ -29,14 +34,14 @@ def render(element_html: str, data: pl.QuestionData) -> str:
 
     hints_to_display = []
 
-    for idx, (priority, hint) in enumerate(sorted(hints)):
+    for idx, (priority, _, hint) in enumerate(sorted(hints), 1):
```

Comment:
For my own understanding: when `sorted` is applied to a list of tuples, it sorts by the first element of each tuple, and considers the second, third, etc. elements for tie-breaking?

Reply:
Yes, that's correct. That was the issue before, it would break ties by alphabetically sorting the contents of the hints. That won't be an issue anymore because the second elements are all distinct (no ties possible).